test/models/test_repr.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+from unittest import TestCase
+import _models
+
+
+# ensure that all models have a __repr__ method implemented
+class TestAllModels(TestCase):
+
+    """
+    ColumnItem wrapper_descriptor
+    ConnectionCredentials wrapper_descriptor
+    DataAccelerationReportItem wrapper_descriptor
+    DatabaseItem wrapper_descriptor
+    DQWItem wrapper_descriptor
+    UnpopulatedPropertyError wrapper_descriptor
+    FavoriteItem wrapper_descriptor
+    FlowRunItem wrapper_descriptor
+    IntervalItem wrapper_descriptor
+    DailyInterval wrapper_descriptor
+    WeeklyInterval wrapper_descriptor
+    MonthlyInterval wrapper_descriptor
+    HourlyInterval wrapper_descriptor
+    BackgroundJobItem wrapper_descriptor
+    PaginationItem wrapper_descriptor
+    Permission wrapper_descriptor
+    ServerInfoItem wrapper_descriptor
+    SiteItem wrapper_descriptor
+    TableItem wrapper_descriptor
+    Resource wrapper_descriptor
+    """
+
+    # not all models have __repr__ yet: see above list
+    @pytest.mark.xfail()
```

Comment:
Is there a work item tracking adding `repr` to the remaining models? Could we enable this test and remove these models from `m` at runtime?

Reply:
https://github.com/tableau/server-client-python/issues/140 but I haven't added much info there. I like the runtime idea, will add.